client/connect-account-page/index.tsx

```diff
@@ -29,7 +29,12 @@ import LogoImg from 'assets/images/woopayments.svg?asset';
 import strings from './strings';
 import './style.scss';
 import InlineNotice from 'components/inline-notice';
-import { WooPaymentMethodsLogos } from 'components/payment-method-logos';
+
+const WooPaymentMethodsLogos = React.lazy( () =>
+	import( '@woocommerce/onboarding' ).then( ( module ) => {
```

Comment:
I'm curious why it has to be imported asynchronously.

Reply:
Oh shoot, just saw the post paJDYF-eod-p2 which answers my question. So, we are lazy loading this component for performance reasons.

Reply:
Yes, correct!